style2.css

```diff
@@ -110,40 +109,41 @@
     font-size: 1.1em;
     font-weight: bold;
 }
-.input {
+form {
     margin-top: 20px;
     padding-bottom: 20px;
 }
-.input input {
+form input {
     width: 100%;
     border: none;
     border-bottom: 1px solid grey;
-    padding: 25px;
+    padding: 25px 25px 25px 10px;
 }
-.input .icon {
+form .icon {
     position: relative;
 }
-.input .icon i {
+form .icon i {
```

Comment:
avoid form .icon i {}, form input{} etc, give a class and write directly (for all cases)